src/main/java/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,78 @@
+package duke.tasks;
+
+import duke.dukeexception.*;
+import duke.messages.*;
+import java.util.ArrayList;
+
+public class TaskList {
```

Comment:
Maybe you should write some lines of comments to explain the idea behind add,delete and so on...quite plain code here.